src/app/services/sort.service.ts

```diff
@@ -0,0 +1,26 @@
+import { Injectable } from '@angular/core';
+import { User } from '../interfaces/interfaces';
+import { StorageService } from './storage.service';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class SortService {
+  store: StorageService
+
+  constructor(store: StorageService) { 
+    this.store = store;
+  }
+
+  sortByName(): User[] {
+    const sortedUsers = this.store.users.sort(
+      (a: User, b: User) => a.name.toLocaleLowerCase() > b.name.toLocaleLowerCase() ? 1 : -1);
```

Comment:
Pls, set more informative names and you can minimize code as `a.name.toLocaleLowerCase() - b.name.toLocaleLowerCase()`